### ISSUE TITLE
ValueTuple reference appeared in VS 15.7

### ISSUE THREAD

**Issue by cmeeren** [Microsoft/visualfsharp#4859]
I just installed VS 15.7, and now my F# projects have a `System.ValueTuple` reference I can't remove, with a yellow triangle.

![image](https://user-images.githubusercontent.com/7766733/39760501-c16f2b32-52d5-11e8-9168-e1f40b08c497.png)

If I try to delete it, I get the following error:

![image](https://user-images.githubusercontent.com/7766733/39760521-cdfbd580-52d5-11e8-90d7-1e759b43f1df.png)

Version info:

```
Microsoft Visual Studio Professional 2017 
Version 15.7.0
VisualStudio.15.Release/15.7.0+27703.1
Microsoft .NET Framework
Version 4.7.02556

Installed Version: Professional

Visual F# Tools 10.1 for F# 4.1   15.7.0.0.  Commit Hash: 16ecf5a30ad868d183c58e4a71a71c23d4ed3ba9.
Microsoft Visual F# Tools 10.1 for F# 4.1
```

**Comment by cartermp** [comment]
Closing as dupe of: https://github.com/Microsoft/visualfsharp/issues/4412

This reference to SVT has always been there - there is an issue in understanding the reference in F# targets. It's benign and doesn't affect anything at runtime, it just manifests as a warning right now.

**Comment by cmeeren** [comment]
Good to know. But are you sure it's the same? The reference wasn't there in VS 15.6, it only appeared after I updated to 15.7, which was just released, whereas #4412 is two months old.

**Comment by cartermp** [comment]
Yes. The first preview of 15.7 shipped alongside 15.6, which is when the issue first appeared.
